<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpcUaClientDialog</class>
 <widget class="QDialog" name="OpcUaClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OPC UA Client Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="opcUaServerNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="disconnectPushButton">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="connectPushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>OPC UA Nodes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="opcUaNodesTreeWidget">
          <attribute name="headerDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>Node</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date time stamp</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <item>
           <property name="text">
            <string notr="true">RBS.BeamSpectrum.01</string>
           </property>
           <item>
            <property name="text">
             <string notr="true">State</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Command</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Value</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ValueIntegral</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">HeartBeat</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpcUaClientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpcUaClientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
